plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -295,9 +295,15 @@ func (p *HelmChartInflationGeneratorPlugin) pullCommand() []string {
 	args := []string{
 		"pull",
 		"--untar",
-		"--untardir", p.absChartHome(),
-		"--repo", p.Repo,
-		p.Name}
+		"--untardir", p.absChartHome()}
+
+	// OCI pull combine the repo and the chart name into one URL
+	if strings.HasPrefix(p.Repo, "oci://") {
+		chartUrl := p.Repo + "/" + p.Name
```

Comment:
Just to clarify, it's saying `Url` should be `URL` in the variable name, I missed the distinction on first read-through.